sdk/identity/identity/src/credentials/managedIdentityCredential.ts

```diff
@@ -200,10 +207,10 @@ export class ManagedIdentityCredential implements TokenCredential {
       }
 
       // If we received any response, the endpoint is available
-      logger.info(`ManagedIdentityCredential: IMDS endpoint is available`);
+      logger.info(`IMDS endpoint is available`);
```

Comment:
I might have missed it but do you print that you're using the MSI_ENDPOINT when IMDS is not available?

Reply:
@catalinaperalta let me know what's your approach and I'll adhere.

Reply:
Hmmm I see. Well could you log that you're using the MSI_ENDPOINT if the first step is successful?

Reply:
I added the logs! Let me know how the PR looks now 🌞